client/asset/eth/eth.go

```diff
@@ -328,6 +329,21 @@ func (w *TokenWallet) Info() *asset.WalletInfo {
 	}
 }
 
+// genWalletSeed uses the wallet seed passed from core as the entropy for
+// generating a BIP-39 mnemonic. Then it returns the wallet seed generated
+// from the mnemonic which can be used to derive a private key.
+func genWalletSeed(entropy []byte) ([]byte, error) {
+	if len(entropy) != 32 {
+		return nil, fmt.Errorf("wallet seed must be 32 bytes long")
+	}
+	mnemonic, err := bip39.NewMnemonic(entropy)
+	defer func() { mnemonic = "" }()
```

Comment:
This is probably ineffectual. Strings can't be forcefully cleared from memory apparently. https://stackoverflow.com/questions/39968084/is-it-possible-to-zero-a-golang-strings-memory-safely